astropy/io/ascii/tests/test_cds_header_from_readme.py

```diff
@@ -228,15 +226,15 @@ def test_cds_no_whitespace():
     data = "data/cds/null/table.dat"
     r = ascii.Cds(readme)
     r.read(data)
-    assert_equal(r.header.cols[6].description, "Temperature class codified (10)")
-    assert_equal(r.header.cols[6].null, "")
-    assert_equal(r.header.cols[7].description, "Equivalent width (in mA)")
-    assert_equal(r.header.cols[7].null, "-9.9")
-    assert_equal(
-        r.header.cols[10].description,
+    assert(r.header.cols[6].description == "Temperature class codified (10)")
+    assert(r.header.cols[6].null == "")
+    assert(r.header.cols[7].description == "Equivalent width (in mA)")
+    assert(r.header.cols[7].null == "-9.9")
+    assert(
+        r.header.cols[10].description ==
         "DAOSPEC quality parameter Q (large values are bad)",
     )
```

Comment:
```suggestion
    assert (
        r.header.cols[10].description
        == "DAOSPEC quality parameter Q (large values are bad)"
    )
```
Here it was probably just `black`'s annoying habit of inserting a trailing comma everywhere that created a spurious tuple here, but otherwise this should work.

Reply:
i have changed these lines should i commit